addons/mail/static/src/discuss/call/public_web/discuss_client_action_patch.js

```diff
@@ -0,0 +1,23 @@
+import { DiscussClientAction } from "@mail/core/public_web/discuss_client_action";
+import { useState } from "@odoo/owl";
+
+import { useService } from "@web/core/utils/hooks";
+import { patch } from "@web/core/utils/patch";
+
+patch(DiscussClientAction.prototype, {
+    setup() {
+        super.setup(...arguments);
+        this.rtc = useState(useService("discuss.rtc"));
+    },
+    async restoreDiscussThread() {
+        await super.restoreDiscussThread(...arguments);
+        const action = this.props.action;
+        if (!action) {
+            return;
+        }
+        const call = action.context?.call || action.params?.call;
+        if (call === "accept") {
+            await this.rtc.toggleCall(this.store.discuss.thread);
+        }
+    },
```

Comment:
It's hard to understand where all of those params are coming from, it's really not obvious what this code does and where the values are coming from.

Also toggle doesn't help (and is probably wrong, if the call is already in progress).